specification/monitor/resource-manager/Microsoft.Insights/preview/2020-01-01-preview/managementGroupDiagnosticSettings_API.json

```diff
@@ -0,0 +1,342 @@
+        "description": "Deletes existing management group diagnostic settings for the specified resource.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ManagementGroupIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/NameParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Successful request to remove a management group diagnostic setting"
+          },
+          "204": {
+            "description": "Successful request to remove a management group diagnostic setting"
+          }
+        },
+        "x-ms-examples": {
+          "Deletes the management group diagnostic setting": {
+            "$ref": "./examples/deleteManagementGroupDiagnosticSetting.json"
+          }
+        }
+      }
+    },
+    "/providers/microsoft.management/managementGroups/{managementGroupId}/providers/microsoft.insights/diagnosticSettings": {
+      "get": {
+        "tags": [
+          "ManagementGroupDiagnosticSettings"
+        ],
+        "operationId": "ManagementGroupDiagnosticSettings_List",
+        "description": "Gets the active management group diagnostic settings list for the specified management group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ManagementGroupIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Successful request to get more information about management group diagnostic setting",
+            "schema": {
+              "$ref": "#/definitions/ManagementGroupDiagnosticSettingsResourceCollection"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets the management group diagnostic setting": {
+            "$ref": "./examples/listManagementGroupDiagnosticSettings.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ManagementGroupProxyOnlyResource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource Id"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource name"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource type"
+        },
+        "location": {
```

Comment:
this resource should not have a location since it is proxyOnly

Reply:
Removed

Reply:
this property is still here